### ISSUE TITLE
output in different audio file formats

### ISSUE THREAD

**Issue by BBC-Esq** [collabora/WhisperSpeech#103]
Can we incorporate the functionality to output the audio to different file formats such as mp3, flas or wma or flac?

**Comment by gongouveia** [comment]
Anyways, you can use FFMPEG to do the trick. You can change the output file from whisperspeech and convert it to any format you want.
`ffmpeg -i input.mpg ... -f mp4 output
`
I will search into it, may create a pr

**Comment by jpc** [comment]
It seems like soundfile should support several formats (at least MP3 and OGG) I think it should figure it out automatically based on the extension?

**Comment by BBC-Esq** [comment]
Sounds file does support multiple audio formats, but there would need to be a parameter to specify the output, and the source code would have to be modified to act in accordance...if we're open to a pull request?  I just didn't see it implemented yet.

**Comment by zoq** [comment]
Please feel free to open a PR that adds the support; I think `ogg, mp3` should be useful for a lot of people.

**Comment by BBC-Esq** [comment]
Yeah, think that I'll try to do that for the ```generate_to_file``` method in pipeline.

**Comment by BBC-Esq** [comment]
I doublechecked the issue using the "minimal" script in the examples folder.  I changed the file name to .flac, .mp3, and .ogg.  It automatically worked for the first two but I couldn't get .ogg to play.  I think the issue is that I don't have a player that supports .ogg because it successfully created the .ogg file.  Therefore, I'm closing out this issue since apparently the ```generate_to_file``` method already supports this functionality!
